go.mod

```diff
@@ -15,63 +15,72 @@ require (
 	go.uber.org/zap v1.24.0
 	golang.org/x/exp v0.0.0-20220921164117-439092de6870
 	golang.org/x/sync v0.1.0
-	k8s.io/api v0.26.0
-	k8s.io/apimachinery v0.26.0
-	k8s.io/client-go v0.26.0
-	sigs.k8s.io/controller-runtime v0.14.1
+	k8s.io/api v0.25.5
+	k8s.io/apimachinery v0.25.5
+	k8s.io/client-go v0.25.5
+	sigs.k8s.io/controller-runtime v0.13.1
```

Comment:
Lesson learned not to update these arbitrarily. 

The control plane of k8s v1.23 and v1.24 can't handle a new alpha field in corev1 Resources. 

```
for: "STDIN": error when patching "STDIN": CustomResourceDefinition.apiextensions.k8s.io "statefuljobs.cosmos.strange.love" is invalid: [spec.validation.openAPIV3Schema.properties[spec].properties[podTemplate].properties[spec].properties[containers].items.properties[resources].properties[claims].items.x-kubernetes-map-type: Invalid value: "null": must be atomic as item of a list with x-kubernetes-list-type=set, spec.validation.openAPIV3Schema.properties[spec].properties[podTemplate].properties[spec].properties[volumes].items.properties[ephemeral].properties[volumeClaimTemplate].properties[spec].properties[resources].properties[claims].items.x-kubernetes-map-type: Invalid value: "null": must be atomic as item of a list with x-kubernetes-list-type=set, spec.validation.openAPIV3Schema.properties[spec].properties[podTemplate].properties[spec].properties[initContainers].items.properties[resources].properties[claims].items.x-kubernetes-map-type: Invalid value: "null": must be atomic as item of a list with x-kubernetes-list-type=set, spec.validation.openAPIV3Schema.properties[spec].properties[podTemplate].properties[spec].properties[ephemeralContainers].items.properties[resources].properties[claims].items.x-kubernetes-map-type: Invalid value: "null": must be atomic as item of a list with x-kubernetes-list-type=set]
```

I'm hoping once the control plane catches up, this won't be a problem. But it could indicate we may need to define our own resources in the future.